hedera-node/hedera-addressbook-service-impl/src/main/java/com/hedera/node/app/service/addressbook/impl/schemas/V053AddressBookSchema.java

```diff
@@ -180,18 +181,6 @@ private Map<Long, NodeAddress> getNodeAddressMap(@NonNull final MigrationContext
      * @return the {@link ServiceEndpoint} object
      */
     public static ServiceEndpoint endpointFor(@NonNull final String host, final int port) {
-        final var builder = ServiceEndpoint.newBuilder().port(port);
-        if (IPV4_ADDRESS_PATTERN.matcher(host).matches()) {
-            final var octets = host.split("[.]");
-            builder.ipAddressV4(Bytes.wrap(new byte[] {
-                (byte) Integer.parseInt(octets[0]),
-                (byte) Integer.parseInt(octets[1]),
-                (byte) Integer.parseInt(octets[2]),
-                (byte) Integer.parseInt(octets[3])
-            }));
-        } else {
-            builder.domainName(host);
-        }
-        return builder.build();
+        return AddressBookUtils.endpointFor(host, port);
```

Comment:
Is a `V053` schema still getting executed?  When would they no longer be executed?

Reply:
This change merely calls the same logic, now contained in `AddressBookUtils`, to construct the service endpoint. 

To answer your question, yes, this schema is still executed for all nodes starting from genesis or < v53 cases.